content/es/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -0,0 +1,751 @@
+está `Running`,
+también puedes ver información sobre cuando el contenedor entró en
+estado `Running`.
+
+### `Terminated` {#container-state-terminated}
+
+Un contenedor en el estado `Terminated` comenzó su ejecución y luego se terminó
+con éxito o falló por alguna razón.
+Cuando usas `kubectl` para consultar un Pod con un contenedor que
+está `Terminated`, puedes ver un motivo, y un código de salida, y la hora de
+inicio y de finalización del contenedor.
+
+Si un contenedor tiene un hook `preStop` configurado, el hook se ejecuta antes
+de que el contenedor entre en estado `Terminated`.
+
+## Cómo los Pods manejan los problemas con los contenedores {#container-restarts}
+
+Kubernetes maneja los fallos de los contenedores dentro de los Pods usando una [política de reinicio, `restartPolicy` en inglés](#restart-policy) definida en la especificación `spec` del Pod.
+Esta política determina cómo reacciona Kubernetes cuando los contenedores salen debido a errores u otras razones, que sigue la siguiente secuencia:
+
+1. **Fallo inicial**: Kubernetes intenta un reinicio inmediato basado en la `restartPolicy` del Pod.
+1. **Fallos repetidos**:
+   Después del fallo inicial, Kubernetes aplica un retraso exponencial para los reinicios subsiguientes, descrito en [restartPolicy](#restart-policy).
+   Esto evita que los intentos de reinicio rápidos y repetidos sobrecarguen el sistema.
+1. **Estado de CrashLoopBackOff**:
+   Esto indica que el mecanismo de retraso exponencial está actualmente en efecto para un contenedor dado que está en un bucle de fallos, fallando y reiniciando repetidamente.
+1. **Reinicio del retraso**:
+   Si un contenedor funciona correctamente durante un cierto período (por ejemplo, 10 minutos), Kubernetes reinicia el retraso, tratando cualquier nuevo fallo como el primero.
+
+2. En la práctica, un `CrashLoopBackOff` es una condición o evento que podría verse como salida del comando `kubectl`, al describir o listar Pods, cuando un contenedor en el Pod no arranca correctamente y luego intenta y falla continuamente en un bucle.
+
+En otras palabras, cuando un contenedor entra en el bucle de fallos, Kubernetes aplica el retraso exponencial mencionado en la [Política de reinicio del contenedor](#restart-policy).
+Este mecanismo evita que un contenedor defectuoso sobrecargue el sistema con intentos de inicio fallidos continuos.
+
+El `CrashLoopBackOff` puede ser causado por problemas como los siguientes:
+
+* Errores de la aplicación que hacen que el contenedor salga.
+* Errores de configuración, como variables de entorno incorrectas o archivos de configuración faltantes.
+* Restricciones de recursos, donde el contenedor puede no tener suficiente memoria o CPU para arrancar correctamente.
+* Fallos en los chequeos de salud si la aplicación no comienza a servir dentro del tiempo esperado.
+* Las sondas de liveness o de arranque del contenedor devuelven un resultado de `Failure` como se menciona en la [sección de sondas](#container-probes).
```

Comment:
This Hugo reference is missing. See the below suggestion: https://github.com/kubernetes/website/pull/47204/files#r1725989838